maxdiff/tests/test_baselines/Test.amxd.txt

```diff
@@ -21,6 +20,7 @@ project:
 		data:
 			collContent.txt
 ----------- objects -----------
+[message 😋] fontsize: 20.0
```

Comment:
Should we test that something like this is properly patch printed if the encoding is configured to something other than UTF-8 in the environment?

Reply:
I think we should ideally, but I don't know how :p I'm ok to leave it like this for now and fix this when an idea comes up.